agent/src/main/java/com/teamscale/jacoco/agent/options/AgentOptions.java

```diff
@@ -79,6 +79,12 @@ public class AgentOptions {
 	 */
 	/* package */ String originalOptionsString;
 
+	/** Whether debug logging is active or not. */
+	/* package */ boolean debugLogging = false;
+
+	/** Explicitly defined log file. */
+	/* package */ String loggingFilePath = "";
```

Comment:
Is this just working for debug logging or for standard logging as well?

Reply:
Only for debug logging

Reply:
Then I would rename to debugLoggingFilePath to make it clear